users/untrusted.rst

```diff
@@ -136,6 +143,19 @@ and receive the data for the new file over the network. However if you have a bi
 e.g. ``video.mp4``, and you modify just a part of it (e.g. video metadata), only
 the changed block is transferred as usual.
 
+Decrypting data
+---------------
+
+Say you recovered a backup of an untrusted device. To get at the plaintext data,
+you can either set up Syncthing to share the data as a receive-only folder to
+a trusted device as explained above. It's also possibly to directly decrypt the
```

Comment:
This part isn't implemented yet as the scanner doesn't import the encrypted fileinfos. Currently offline decryption is the only option if the database is gone.